dbbackup/settings.py

```diff
@@ -1,18 +1,22 @@
 # DO NOT IMPORT THIS BEFORE django.configure() has been run!
 
 import os
+import tempfile
+import socket
 import warnings
 from django.conf import settings
 
 DATABASES = getattr(settings, 'DBBACKUP_DATABASES', list(settings.DATABASES.keys()))
 
 # Fake host
-DBBACKUP_FAKE_HOST = getattr(settings, 'DBBACKUP_FAKE_HOST', 'django-dbbackup')
+HOSTNAME = getattr(settings, 'DBBACKUP_HOSTNAME', socket.gethostname())
```

Comment:
I changed ``DBBACKUP_FAKE_HOST``to ``HOSTNAME`` which is more logical for users.
I also use ``socket.gethostname`` for get default